src/IK/include/BipedalLocomotion/IK/SE3Task.h

```diff
@@ -77,6 +86,7 @@ class SE3Task : public System::LinearTask
      * |            `frame_name`            | `string` |                       Name of the frame controlled by the SE3Task                      |    Yes    |
      * |             `kp_linear`            | `double` |                             Gain of the position controller                            |    Yes    |
      * |            `kp_angular`            | `double` |                           Gain of the orientation controller                           |    Yes    |
+     * |               `mask`               | `vector<bool>` |  Mask representing the linear DoFs controlled. E.g. [T,F,T] will enable the control on the x and z coordinates only and the angular part. (Default value, [T,T,T])   |    No    |
```

Comment:
I would suggest using ``1`` and ``0`` instead of ``T`` and ``F``. If someone blindly copies it in a conf file, I am not sure that ``yarp`` will interpret ``F`` as ``false``.